src/utilities/getSmallestNumber.js

```diff
@@ -0,0 +1,6 @@
+export function getSmallestNumber(arrayOfNumbers) {
+  if (arrayOfNumbers.length === 0) {
+    return undefined;
```

Comment:
That's a rather surprising choice to return undefined explicitly. Usually, in JavaScript, if you want to explicitly signify an absence of a value, you return null.

When I said that this function would probably return undefined for an empty array, I thought that you might use a for loop to solve this exercise.

I suggest you use `return null` instead.